### PR TITLE
add social.whalebird.WhalebirdDesktop

### PR DESCRIPTION
**Pull request by Lunarequest**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. 
- https://github.com/h3poteto/whalebird-desktop/issues/858
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Lunarequest** [comment]
bot, build social.whalebird.WhalebirdDesktop

**Comment by Lunarequest** [comment]
bot, build social.whalebird.WhalebirdDesktop

File: social.whalebird.WhalebirdDesktop.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=/run/dbus:ro
```
Comment (hfiguiere) [inline social.whalebird.WhalebirdDesktop.yaml:19]: why do you need this one?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: social.whalebird.WhalebirdDesktop.yaml (hunk 1 above)
Comment (Lunarequest) [reply social.whalebird.WhalebirdDesktop.yaml:19]: I'd like to drop it but for some reason the app uses the system bus and has a weird error in the console without it

**Comment by Lunarequest** [comment]
bot, build social.whalebird.WhalebirdDesktop

File: social.whalebird.WhalebirdDesktop.yaml (hunk 1 above)
Comment (hfiguiere) [reply social.whalebird.WhalebirdDesktop.yaml:19]: what kind of weird error?

Comment (Lunarequest) [reply social.whalebird.WhalebirdDesktop.yaml:19]: [3:1116/062432.017970:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory

**Comment by nedrichards** [comment]
bot, build social.whalebird.WhalebirdDesktop

File: social.whalebird.WhalebirdDesktop.desktop [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```
Comment (barthalion) [inline social.whalebird.WhalebirdDesktop.desktop:1]: Should br submitted upstream.

File: social.whalebird.WhalebirdDesktop.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline social.whalebird.WhalebirdDesktop.metainfo.xml:1]: Should br submitted upstream.

File: social.whalebird.WhalebirdDesktop.yaml (hunk 1 above)
Comment (barthalion) [reply social.whalebird.WhalebirdDesktop.yaml:20]: You should use `--socket=system-bus --log-system-bus` to find what it tries to access.

Comment (Lunarequest) [reply social.whalebird.WhalebirdDesktop.yaml:20]: It looks like it doesn't access anything through the system bus, should I just ignore the error?

File: social.whalebird.WhalebirdDesktop.desktop (hunk 2 above)
Comment (Lunarequest) [reply social.whalebird.WhalebirdDesktop.desktop:1]: Files have been upstreamed

**Comment by barthalion** [comment]
bot, build social.whalebird.WhalebirdDesktop

**Pull request closed without merging**